<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationDataWidgetControls</class>
 <widget class="QWidget" name="RegistrationDataWidgetControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>150</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="widgetList">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QmitkSingleNodeSelectionWidget" name="imageSelection" native="true"/>
     </item>
     <item>
      <widget class="QSpinBox" name="index"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QmitkSingleNodeSelectionWidget" name="imageMaskSelection" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="referencePointSetData">
     <item>
      <widget class="QPushButton" name="addMovingPointSet">
       <property name="minimumSize">
        <size>
         <width>25</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QmitkSingleNodeSelectionWidget" name="pointSetSelection" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QToolButton" name="btnSaveTransforms">
       <property name="toolTip">
        <string>Save elastix transformation files</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../mitk/Plugins/org.mitk.gui.qt.ext/resources/org_mitk_icons.qrc">
         <normaloff>:/org_mitk_icons/icons/awesome/scalable/actions/document-save.svg</normaloff>:/org_mitk_icons/icons/awesome/scalable/actions/document-save.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnLoadPreset">
       <property name="toolTip">
        <string>Open elastix transformation files</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../mitk/Plugins/org.mitk.gui.qt.ext/resources/org_mitk_icons.qrc">
         <normaloff>:/org_mitk_icons/icons/awesome/scalable/actions/document-open.svg</normaloff>:/org_mitk_icons/icons/awesome/scalable/actions/document-open.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnApplyTransforms">
       <property name="toolTip">
        <string>Apply transformations to selected image and attachments</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../mitk/Plugins/org.mitk.gui.qt.ext/resources/org_mitk_icons.qrc">
         <normaloff>:/org_mitk_icons/icons/awesome/scalable/actions/edit-redo.svg</normaloff>:/org_mitk_icons/icons/awesome/scalable/actions/edit-redo.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnRemove">
       <property name="toolTip">
        <string>Delete this moving entity</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../mitk/Plugins/org.mitk.gui.qt.ext/resources/org_mitk_icons.qrc">
         <normaloff>:/org_mitk_icons/icons/awesome/scalable/actions/edit-delete.svg</normaloff>:/org_mitk_icons/icons/awesome/scalable/actions/edit-delete.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>28</width>
         <height>28</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkSingleNodeSelectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">QmitkSingleNodeSelectionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../mitk/Plugins/org.mitk.gui.qt.ext/resources/org_mitk_icons.qrc"/>
 </resources>
 <connections/>
</ui>
